debian/rules

```diff
@@ -145,10 +103,14 @@ override_dh_install:
 	install --mode=0644 debian/*.socket debian/snapd/$(SYSTEMD_UNITS_DESTDIR)
 	install --mode=0644 debian/snapd.service debian/snapd/$(SYSTEMD_UNITS_DESTDIR)
 	install --mode=0644 debian/*.target debian/snapd/$(SYSTEMD_UNITS_DESTDIR)
-ifeq ($(RELEASE),trusty)
-	dh_link debian/snapd/$(SYSTEMD_UNITS_DESTDIR)/snapd.service debian/snapd/$(SYSTEMD_UNITS_DESTDIR)/multi-user.target.wants/snapd.service
-	dh_link debian/snapd/$(SYSTEMD_UNITS_DESTDIR)/snapd.autoimport.service debian/snapd/$(SYSTEMD_UNITS_DESTDIR)/multi-user.target.wants/snapd.autoimport.service
-endif
+	install --mode=0644 debian/snapd.boot-ok.service debian/snapd/$(SYSTEMD_UNITS_DESTDIR)
+	install --mode=0644 debian/snapd.firstboot.service debian/snapd/$(SYSTEMD_UNITS_DESTDIR)
+	dh_link /$(SYSTEMD_UNITS_DESTDIR)/snapd.firstboot.service /$(SYSTEMD_UNITS_DESTDIR)/multi-user.target.wants/snapd.firstboot.service
```

Comment:
instead of manually shipping the symlinks I'd much rather you used `systemctl enable`. You can even `systemctl enable --now $UNIT` to start it at the same time as you enable it.